src/index.js

```diff
@@ -16,7 +16,49 @@ import leaveList from './modifiers/leaveList';
 import insertText from './modifiers/insertText';
 import createLinkDecorator from './decorators/link';
 import createImageDecorator from './decorators/image';
+import { addText, addEmptyBlock } from './utils';
 
+const INLINE_STYLE_CHARACTERS = [' ', '*', '_'];
+
+function checkCharacterForState(editorState, character) {
+  if (INLINE_STYLE_CHARACTERS.indexOf(character) === -1) { return editorState; }
```

Comment:
Removed it. That check was redundant. currently, checkCharacterForState is only called with one of INLINE_STYLE_CHARACTERS. Now code coverage is 100%.